src/main.ts

```diff
@@ -48,8 +131,8 @@ export async function run(): Promise<void> {
     core.endGroup()
 
     // Create branch new branch
-    core.startGroup(`Create new branch from ${inputs.branch}`)
-    await gitExecution(['checkout', '-b', prBranch, `origin/${inputs.branch}`])
+    core.startGroup(`Create new branch from ${branch}`)
```

Comment:
branchs --> branches

Reply:
Good catch, I think that was in the original source.